### PR TITLE
fix: refresh button refetches courses

### LINKED ISSUE #736: Refresh button doesn't do anything
Try it yourself.

### PR DESCRIPTION
**Pull request by ap0nia**
## Description
Refresh button re-fetches courses

## Issues

Closes #736

### CODE REVIEW

**Review by EricPedley — APPROVED** [review]

**Review by MinhxNguyen7 — CHANGES_REQUESTED** [review]
Still nothing seems to happen when I click the refresh button. Not sure if I'm missing something

**Comment by ap0nia** [comment]
There were multiple websoc caches in the app 🤦

File: apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx [hunk 1]
```diff
@@ -202,8 +203,11 @@ export function CourseRenderPane() {
     }, []);
 
     useEffect(() => {
+        console.log('loading courses, cache: ', websocCache);
         loadCourses();
-    }, []);
+    }, [props.id]);
+
+    console.log('id: ', props.id);
```
Comment (KevinWu098) [inline apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx]:
```suggestion
```

Living rent-free 🤭

**Review by KevinWu098 — COMMENTED** [review]
LGTM otherwise

**Review by MinhxNguyen7 — APPROVED** [review]
LGTM!

I think the multiple caches might've been me. I vaguely remember extracting part of `helpers` into `course-helpers`, but I probably forgot to delete the old stuff. I'll make an issue for that.

**Comment by EricPedley** [comment]
Bruh why does it still say there are conflicts 😭

**Review by EricPedley — APPROVED** [review]
approving my own additions to the PR to give me the green merge button (?)

**Comment by EricPedley** [comment]
needs another review for nice green merge button

**Review by KevinWu098 — APPROVED** [review]
Trusting @EricPedley on this one! Gonna leave the final merge to someone else though, since I haven’t looked at the code much 🤭

**Pull request merged**